File: app/src/main/java/com/texon/engineeringsmartbook/view/ui/auth/OtpVerify.java
package com.texon.engineeringsmartbook.view.ui.auth;

import android.content.Intent;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

import com.texon.engineeringsmartbook.R;

public class OtpVerify extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_otp_verify);
    }

    @Override
    public void onBackPressed() {
        Intent intent = new Intent(getApplicationContext(), SignUp.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/texon/engineeringsmartbook/view/ui/auth/OtpVerify.kt
package com.texon.engineeringsmartbook.view.ui.auth

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.content.Intent
import com.texon.engineeringsmartbook.databinding.ActivityOtpVerifyBinding

class OtpVerify : AppCompatActivity() {
    private lateinit var binding: ActivityOtpVerifyBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityOtpVerifyBinding.inflate(layoutInflater)
        setContentView(binding.root)
    }

    override fun onBackPressed() {
        val intent = Intent(applicationContext, SignUp::class.java)
        startActivity(intent)
        finish()
    }
}